### ISSUE TITLE
List Out

### ISSUE THREAD

**Issue by Howler4695** [KeeslerProgrammers/CACAttack#21]
List of members who didn't meet CAC-IN timeline

**Comment by ArchonDais** [comment]
where should List-In/List-Out be written?

**Comment by Howler4695** [comment]
You mean like the scope of the list or the classes the list will be referencing?

**Comment by ArchonDais** [comment]
well I suppose first of all:

Should the List be its own class? if that's the case, self contained methods to make changes to that list would be ideal.

if not, is the list is generated from a more general student roster (because the list that one CQ uses will not be the same as the other)

**Comment by ArchonDais** [comment]
for v0.1 we only need one list, but if we can figure that out now it will save us effort to change later

**Comment by Howler4695** [comment]
Yeah, general student class, then just make the methods outlining how the students will go in and out. We can make a list with All of A Bay on it and use it as if we're reading from of DB (for testing.)
